console

```diff
@@ -42,27 +73,103 @@ function setup {
 		echo "[OK] kubectl found"
 	fi
 
+    read -p "What is the name of the context? " CONTEXT_NAME
+    read -p "What is the IP of the cluster? " CLUSTER_IP
+    read -p "What is the username? " USERNAME
+    read -p "What is the password? " PASSWORD
+    read -p "What is the default namespace you want to use? " NAMESPACE
+
+    kubectl config set-credentials $CONTEXT_NAME-$USERNAME --username=$USERNAME --password=$PASSWORD
+    kubectl config set-cluster $CONTEXT_NAME --server=https://$CLUSTER_IP --insecure-skip-tls-verify=true
+    kubectl config set-context $CONTEXT_NAME --cluster=$CONTEXT_NAME --user=$CONTEXT_NAME-$USERNAME
+    kubectl config use-context $CONTEXT_NAME
 
+    echo "Created the context " $CONTEXT_NAME
+    echo
+    echo "Run 'kubectl cluster-info' to verify the connectivity."
+    echo
 }
 
-while [[ $# -gt 0 ]]; do
-	case $1 in
-	    -h|--help|help)
-	    help
-	    exit 0
-	    ;;
-	    setup)
-	    COMMAND=$1
-	    ;;
-	    *)
-	        echo "Unknown option: "$1
-	    ;;
-	esac
+function setup_remote {
+    REMOTE_DIR="$(dir)/.remote"
 
-	shift
-done
+    read -p "What is the namespace you are using? " NAMESPACE_NAME
+    echo $NAMESPACE_NAME > $REMOTE_DIR
+
+    read -p "What is your remote environment branch you are using? " REMOTE_BRANCH
```

Comment:
"What is the name of the Git branch you are using for you remote environment?" ?